pip_install_privates/install.py

```diff
@@ -18,13 +18,27 @@
     from pip import status_codes, main as pip_main
 
 
-def convert_url(url, token):
+def convert_url_with_token(url, token):
     if url.startswith('git+ssh://git@github.com/'):
         return 'git+https://{}:x-oauth-basic@github.com/{}'.format(token, url[25:])
     elif url.startswith('git+git@github.com:'):
         return 'git+https://{}:x-oauth-basic@github.com/{}'.format(token, url[19:])
     return url
 
+
+def convert_editable_url_with_token(url, token):
+    url = convert_url_with_token(url, token)
+    return ['-e', url]
+
+
+def convert_editable_url(url):
+    if url.startswith('git+ssh://git@github.com/'):
+        url = 'git+https://github.com/{}'.format(url[25:])
```

Comment:
maybe clarify here with a comment why 25 and 19, otherwise put those in constants with a meaningful name or derive them from the string and put that in a var instead:
```
>>> len('git+ssh://git@github.com/')
25
```